docs/doc/30-reference/30-sql/30-query-operators/operators-subquery.md

```diff
@@ -4,11 +4,54 @@ description:
   A subquery is a query nested within another query.
 ---
 
-This topic provides reference information about the subquery operators supported in Databend.
+A subquery is a query nested within another one. Databend supports the following subquery types:
 
-A subquery is a query nested within another query.
+- [Scalar Subqueries](#scalar-subqueries)
+- [EXISTS / NOT EXISTS](#exists--not-exists)
 
-## [ NOT ] EXISTS
+## Scalar Subqueries
+
+A scalar subquery selects only one column or expression and returns only one row at most. A SQL query can have scalar subqueries in any places where a column or expression is expected.
+
+- If a scalar subquery returns 0 rows, Databend will use NULL as the subquery output.
+- If a scalar subquery returns more than one row, Databend will throw an error.
+
+### Example
```

Comment:
I think we can give a more common example, not use **scalar subquery** in `insert` clause, but in `select` clause.